src/main/java/com/autotune/analyzer/serviceObjects/Converters.java

```diff
@@ -61,19 +61,11 @@ public static KruizeObject convertCreateExperimentAPIObjToKruizeObject(CreateExp
                 List<KubernetesAPIObject> kubernetesAPIObjectsList = createExperimentAPIObject.getKubernetesObjects();
                 for (KubernetesAPIObject kubernetesAPIObject : kubernetesAPIObjectsList) {
                     K8sObject k8sObject = null;
-                    // Verify the experiment type.
-                    // If the experiment type is null, default is container type experiment.
-                    // TODO: Update to make this field mandatory and validate if it is a container type.
-                    if (null == kubernetesAPIObject.getExperimentType() || kubernetesAPIObject.getExperimentType().equalsIgnoreCase(AnalyzerConstants.ExperimentTypes.CONTAINER_EXPERIMENT)) {
+                    // check if exp type is null to support remote monitoring experiments
+                    if (null == createExperimentAPIObject.getExperimentType() || createExperimentAPIObject.getExperimentType().equalsIgnoreCase(AnalyzerConstants.ExperimentTypes.CONTAINER_EXPERIMENT)) {
```

Comment:
To avoid using multiple if-else blocks, you can refactor the logic into a more modular approach by leveraging polymorphism, a strategy pattern, or separating out the logic into methods.

private K8sObject createContainerExperiment(KubernetesAPIObject kubernetesAPIObject) { }


private K8sObject createNamespaceExperiment(KubernetesAPIObject kubernetesAPIObject) { }